common/config/rush/pnpm-lock.yaml

```diff
@@ -35,7 +35,7 @@ packages:
     dev: false
     resolution:
       integrity: sha512-wP2Jw6uPp8DEDy0n4KNidvwzDjyVV2xnycEIq7nPzj1rHyb/r+t3OPeNT1INZePP2wy5ZqlwyuyOMTi0ePyY1A==
-  /@azure/amqp-common/1.0.0-preview.10:
```

Comment:
Can you check that you're merging correctly? We're not using amqp-common anymore so I'm not sure why it should get bumped.

Reply:
You're too quick, I'm still updating locally :)